package.json

```diff
@@ -45,15 +45,13 @@
     "monaco-editor": "^0.52.0",
     "prettier": "^3.0.0",
     "react": "^19.0.0",
+    "react-dom": "^19.0.0",
     "rimraf": "^5.0.1",
     "typescript": "^5.1.6"
   },
   "peerDependencies": {
-    "@types/react": ">=16 <= 18",
     "monaco-editor": "^0.52.0",
-    "react": ">=16 <= 18"
-  },
-  "dependencies": {
-    "prop-types": "^15.8.1"
+    "react": "^16.8.0 || ^17.0.0 || ^18.0.0 || ^19.0.0",
```

Comment:
Why did you make it a dependency rather than a peer dependency? Iirc there were some issues with that. 

Also, why not use the range constraint instead of listing versions?

Reply:
The diff is misleading, these are peer dependencies. The dependencies key has gone away altogether.

We can use range constraints here, I think it's just a matter of preference.

Reply:
Ahh, right. 

Let's use range since it's a bit faster to read imo.

Reply:
Done!